docs/security/cosmosdb-jupyter-notebook-2021-08-12.md

```diff
@@ -0,0 +1,67 @@
+
+# Content
+This document describes the mitigation of a security issue described [here](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/).
+
+# Prerequisite
+The Industrial-IoT Platform has been deployed as described in the [Industrial-IoT github repository](https://github.com/Azure/Industrial-IoT/blob/main/docs/deploy/readme.md).
+The steps below refers to a new full installation of the Industrial-IoT Platform with the simulation and Engineering Tool deployed.
+You can also use your own OPC UA server to validate proper operation if the simulation has not been deployed. In case the Engineering Tool has not been deployed, you need to use the OpenAPI (swagger) description of the Industrial-IoT Platform to validate proper operation.
+
+# Validate proper operation
+- Open the "Engineering Tool" of the Industrial-IoT Platform of the deployment.
+- Goto "Discovery" and "Turn on" "Scanning" on linuxgateway, if it is not already on.
+On a new deployment this should result in:
+[09/05/2021 16:11:01] linuxgateway0-lbwvpab_module_discovery: Completed.
+[09/05/2021 16:11:00] linuxgateway0-lbwvpab_module_discovery: 3/3: ... 3 servers found - 2 endpoints found on opc.tcp://linuxgateway0-lbwvpab-sim0.internal.cloudapp.net:51201/...
+
+- Goto "Browse" and "Turn on" "Activation Action" on "opc.tcp://10.1.8.5:51200/
+Should result in:
+opc.tcp://10.1.8.5:51200/		SignAndEncrypt		Basic256Sha256		106		Disconnected		Activated		Turn off		Published Nodes
+
+- Select "opc.tcp://10.1.8.5:51200/" in the Engineering Tool and browse to "/Root/Objects/Server/ServerStatus" and "Turn on" publishing of "CurrentTime"
+Should result in:
+The value of "Current time" in the UI is changing.
+
+- After validated the changing value, "Turn off" publishing in the Engineering Tool.
+
+# Change the Azure Cosmos DB accounts Primary Read-write key
+- Log into the Azure Portal
+
+- Goto the "Azure Cosmos DB account" resource of the resource group into which the Industrial-IoT Platform has been deployed. 
+
+- Goto "Keys" "Read-write Keys" and "Regenerate Primary Key". Fetch the new "Primary Connection String"
+
+- Goto the "Key vault" of the resource group into which the Industrial-IoT Platform has been deployed.
```

Comment:
Vault

Reply:
i used the type name of the portal. i changed it since the upper v feels more natural and is used heavily in the doc.